sys-cluster/kube-router/files/kube-router.confd

```diff
@@ -9,3 +9,5 @@
 # RUN_ROUTER=false
 
 EXTRA_ARGS=''
+
+#KUBE_ROUTER_HEALTHCHECK_URI=localhost:20244/healthz
```

Comment:
Why do you add this here? Maybe add a comment above explaining what it does. Should it be wrapped in `" "`?

Reply:
added a comment and `""`